backend/api/v1/acl.go

```diff
@@ -111,10 +111,6 @@ func (in *ACLInterceptor) getUser(ctx context.Context) (*store.UserMessage, erro
 		return nil, status.Errorf(codes.PermissionDenied, "the user %v has been deactivated by the admin.", principalID)
 	}
 
-	// If RBAC feature is not enabled, all users are treated as OWNER.
-	if !in.licenseService.IsFeatureEnabled(api.FeatureRBAC) {
```

Comment:
I think we can keep this, and only update the `FeatureMatrix`?

Reply:
SG